python/fastexcel/_fastexcel.pyi

```diff
@@ -18,10 +23,42 @@ class _ExcelSheet:
 class _ExcelReader:
     """A class representing an open Excel file and allowing to read its sheets"""
 
-    def load_sheet_by_name(self, name: str) -> _ExcelSheet:
-        """Loads a sheet by name"""
-    def load_sheet_by_idx(self, idx: int) -> _ExcelSheet:
-        """Loads a sheet by index"""
+    def load_sheet_by_name(
+        self,
+        name: str,
+        header_row: None | int = 0,
+        column_names: None | List[str] = None,
+    ) -> _ExcelSheet:
+        """Loads a sheet by name.
+        The header_row (row used for the column labels) is by default the first row.
+        but another value for the header line can be specified.
+        If `None` is passed it means that the sheet doesn't have any header.
+        Column names can override headers found in the document.
+        If column_names is used header_row will be ignored"""
```

Comment:
```suggestion
        """Loads a sheet by name.
        
        The header_row (row used for the column labels) is by default the first row.
        but another value for the header line can be specified.
        If `None` is passed it means that the sheet doesn't have any header.
        Column names can override headers found in the document.
        If column_names is used header_row will be ignored.
        """
```

See conventions for python docstrings: https://peps.python.org/pep-0257/#multi-line-docstrings